### DISCUSSION TITLE
APK Editor Studio v1.7.0

### DISCUSSION THREAD

**Discussion by kefir500** [kefir500/apk-editor-studio#93] [Releases]
- Added automatic Apktool updater.
- Added framework manager.
- Added global Find and Replace.
- Added code search case sensitivity option.
- Added code search by regular expression option.
- Added syntax definition downloader.
- Added ability to open resources in an external app.
- Added ability to disassemble only main DEX classes.
- Added ability to disassemble without debug info.
- Added progress indication for APK cloning.
- Added Android 13 support.
- Added Swedish translation.
- Updated default Apktool to 2.7.0.
- Updated Android SDK Build Tools to 33.0.1.
- Fixed error while saving the currently open resource.

<hr /><em>This discussion was created from the release <a href='https://github.com/kefir500/apk-editor-studio/releases/tag/v1.7.0'>APK Editor Studio v1.7.0</a>.</em>

**Comment by ykapoor** [comment]
I tried using the studio for modifying an app to change some strings in the app from Chinese to English. However, the new apk installs but doesn't run as expected. I tried simply opening the apk and repacking it without any change but it still doesn't work .[App apk link](https://dudu-lucky.com/#%E5%85%B3%E4%BA%8E%E6%8E%88%E6%9D%83)

**Reply by kefir500** [reply]
Hi,
You can try the latest [APK Editor Studio v1.7.1](https://github.com/kefir500/apk-editor-studio/releases/tag/v1.7.1) and see if it fixes your issue.

**Comment by 283365655** [comment]
Thank you so much for your great work, writing great software.
I have a suggestion:
Open apk and install apk on the main interface should not be grayed out after the operation, if it can be operated like a new window, it will bring greater convenience.
Thanks again.
